common/lib/xmodule/xmodule/x_module.py

```diff
@@ -18,9 +18,9 @@
 from webob.multidict import MultiDict
 from lazy import lazy
 
-from xblock.core import XBlock, XBlockAside
+from xblock.core import XBlock, XBlockAside, XBlockMixin
 from xblock.fields import (
-    Scope, Integer, Float, List, XBlockMixin,
```

Comment:
Ah - I was thinking an `__init__` method of a class, not an '__init__.py` file of a module. Thanks.